[DOC-3428] Update default settings topic (#4070)
harness/developer-hub: docs/platform/settings/default-settings.md

* Deployments
* Builds

> ### Required permissions

Reviewer: ```suggestion
## Required permissions
```

Make sure you have the **view** and **edit** permissions for Default Settings.

## Manage Harness Platform default settings

Default Settings include configurable module-specific parameters that you can customize based on your needs, such as enabling or disabling features at specific scopes. 

To access the Default Settings at the Account scope, do the following:

1. In your Harness Account, go to **Account Resources**.

   ![](./static/default-settings-00.png)

2. Select **Default Settings**. The **Account Default Settings** appear.

To access the Default Settings at the Org scope, do the following:

1. In Harness, select your organization. The organization page opens.

2. Select **Default Settings**. The **Account Default Settings** appear.

To access the Default Settings at the Project scope, do the following:

1. In Harness, select your project. 

> 2. Under **PROJECT SETUP**, select **Default Settings**. The **Account Default Settings** appear.

Reviewer: Optional: I think this would look nice with tabs, but it's not necessary.

Author: @aimurphy Agree, added tabs.

## Account scope default settings

> This topic lists Default Settings at the Account scope. You can also configure Default Settings at the Org and Project scopes.

Reviewer: ```suggestion
This topic describes the Default Settings available at the Account scope. You can also configure similar Default Settings at the Org and Project scopes.
```

Optional: Maybe some additional words to suggest that the settings are the same (or mostly the same) at the org and project scopes. Without manually checking, my thinking is that some only apply at the account, so those wouldn't be shown at the org and proj. However, we don't need to specifically enumerate them now. This is a great improvement.

Author: @aimurphy Most settings are at the Account scope. The orgs and projects I checked have a lot less settings. From what I can tell, there aren't any settings in org or proj not covered by available acct settings.

- **Execute Triggers With All Collected Artifacts or Manifests:** Enable this setting to allow all artifacts or manifests collected during the polling interval to trigger a deployment, with one deployment triggered for each artifact/manifest collected. For more information, go to [Trigger pipelines on a new artifact](/docs/platform/triggers/trigger-on-a-new-artifact/).

### Cloud Cost Management

For more information about these settings, go to [Set up perspective preferences](/docs/cloud-cost-management/use-ccm-cost-reporting/ccm-perspectives/perspective-preferences/#default-settings-for-perspective-preferences) and [View and apply recommendations](/docs/cloud-cost-management/use-ccm-cost-optimization/ccm-recommendations/home-recommendations/).

### Notifications

For more information about these settings, go to [Notification settings](/docs/platform/notifications/notification-settings).

### Harness AI Development Assistant

Enable this setting to use Harness AI Development Assistant (AIDA). For more information, go to [Overview of Harness AI Development Assistant](/docs/platform/harness-aida/aida-overview/).

> ### View and Edit Default Settings

Reviewer: ```suggestion
## View and Edit Default Settings
```

It would make sense to combine this with "Manage Harness Platform default settings" or to move it (as a level 3 heading) under that section.

Author: @aimurphy moved under Manage...
